src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -203,6 +203,15 @@ export default function formReducer(state = initialState, action) {
         actionData = unset('vaFacility', actionData);
       }
 
+      // reset community care provider if type of care changes
+      if (
+        getTypeOfCare(actionData)?.id !== getTypeOfCare(state.data)?.id &&
+        (state.pages.ccPreferences || !!state.data.communityCareProvider?.id)
+      ) {
+        newPages = unset('ccPreferences', newPages);
```

Comment:
I needed to clear the schema, too, because otherwise we get two renders when you get to the provider selection page, which messes up the logic we have for setting focus.